dex/src/main/scala/com/wavesplatform/dex/tool/Checker.scala

```diff
@@ -165,6 +165,23 @@ class Checker(superConnector: SuperConnector) {
     val submitted = mkMatcherOrder(assetPairInfo.assetPair, SELL)
     val submittedId = submitted.id()
 
+    def checkEmptyListOfActiveOrders(): ErrorOr[Boolean] =
+      for {
+        _ <- dexRest.getActiveOrdersByPair(env.matcherKeyPair, assetPairInfo.assetPair)
+          .map {
+            case left: Seq[JsValue] => left.foreach { e =>
```

Comment:
```scala
def getActiveOrdersByPair(keyPair: KeyPair, assetPair: AssetPair): ErrorOr[Seq[JsValue]]
type ErrorOr[A] = Either[String, A]
```

So you'll receive either `Left`, or `Right`, but not just `Seq`

Reply:
fixed